docs/chatterino-api.service

```diff
@@ -12,6 +12,7 @@ Environment="CHATTERINO_API_CACHE_TWITCH_CLIENT_ID=XXXXXXXXXXXXXXXXXXXXXXXX"
 Environment="YOUTUBE_API_KEY=XXXXXXXXXXXXXXX"
 Environment="CHATTERINO_API_TWITTER_BEARER_TOKEN=XXXXXXXXXXXXXXXXXXXXXXXXXXX"
 Environment="CHATTERINO_API_IMGUR_CLIENT_ID=XXXXXXXXXXXXXXXXXXXXXXXXXXX"
+Environment=CHATTERINO_API_MAX_CONTENT_SIZE=X
```

Comment:
You should rename `MAX_CONTENT_SIZE` to reflect the "length" part, maybe `MAX_CONTENT_LENGTH` instead